src/assets/scripts/aircraft/AircraftConflict.js

```diff
@@ -124,14 +124,16 @@ export default class AircraftConflict {
      * Check for collision
      */
     checkCollision() {
-        if (this.aircraft[0].wow() || this.aircraft[1].wow()) {
+        if (this.aircraft[0].isOnGround() || this.aircraft[1].isOnGround()) {
```

Comment:
👍 I like this name a lot better.  doesn't leave anything to the imagination about what its for.